recipes/junifer/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "junifer" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/junifer-{{ version }}.tar.gz
+  sha256: 9244c954433e97858868b09d06adfdf20d238d96f1f7f5c276d712ab69e438a2
+
+build:
+  skip: true  # [py<38]
```

Better to make this noarch:
```suggestion
  noarch: python
```